bundles/org.openhab.io.homekit/pom.xml

```
@@ -22,7 +22,7 @@
     <dependency>
       <groupId>io.github.hap-java</groupId>
       <artifactId>hap</artifactId>
-      <version>2.0.1</version>
+      <version>2.0.2-SNAPSHOT</version>
```

```suggestion
      <version>2.0.2</version>
```

probably? Seems released:
https://github.com/hap-java/HAP-Java/releases/tag/hap-2.0.2